e2e/testdata/simple-bundle.json.golden

```diff
@@ -5,118 +5,125 @@
 	"maintainers": [
 		{
 			"name": "John Developer",
-			"email": "john.dev@acmecorp.cool",
-			"url": ""
+			"email": "john.dev@acmecorp.cool"
```

Comment:
super pedantic: `cool` is (apparently) a real TLD. `acmecorp` is not currently registered there but there's no real reason it couldn't be (then pity poor `john.dev` ;-)).

TL;DR: We should use [RFC2606](https://tools.ietf.org/html/rfc2606) domains for any place holder domain name.

Reply:
Is `domain.test` ok 👍 ?

Reply:
Section 2 of https://tools.ietf.org/html/rfc2606 says so, yes. The other common one to use is `example.{net,org,com}` which are reserved in section 3.